### ISSUE TITLE
Failed to spawn: unable to find process with name 'com.androidpentesting.securestore'

### ISSUE THREAD

**Issue by Abhishek1166** [frida/frida#1988]

**Comment by AYiXi** [comment]
https://github.com/sensepost/objection/issues/273#issuecomment-1174823938

**Comment by nareshrathnam** [comment]
I'm also getting this error while switching from `frida 14.x` to `frida 15.x / 16.x`. Need Help.
```
frida --device 241ca882 com.an.biometric.sample -l "fingerprint-bypass.js"

```
![image](https://user-images.githubusercontent.com/114155143/196863797-3205ddea-d9e2-40b0-8fb2-9bc0c100c7e5.png)

**Comment by nareshrathnam** [comment]
@AYiXi

**Comment by AYiXi** [comment]
> @AYiXi



> I'm also getting this error while switching from `frida 14.x` to `frida 15.x / 16.x`. Need Help.
> 
> ```
> frida --device 241ca882 com.an.biometric.sample -l "fingerprint-bypass.js"
> ```
> 
> ![image](https://user-images.githubusercontent.com/114155143/196863797-3205ddea-d9e2-40b0-8fb2-9bc0c100c7e5.png)

do you see https://github.com/sensepost/objection/issues/273#issuecomment-1174823938?
check your package name?
use python to run this not command line.

**Comment by nareshrathnam** [comment]
yeah I seen @AYiXi . I need to run through command line only. Any other methods?

**Comment by AYiXi** [comment]
> 

edit source code, print(`all package name`), and check it
can you use python?

**Comment by nareshrathnam** [comment]
not that well..

**Comment by nareshrathnam** [comment]
can you help? which source code?
